src/resolve/resolved-dns-scope.c

```diff
@@ -1393,3 +1391,31 @@ int dns_scope_remove_dnssd_services(DnsScope *scope) {
 
         return 0;
 }
+
+bool dns_scope_has_route_only_domains(DnsScope *scope) {
+
+        DnsSearchDomain *domain, *first;
+        bool route_only = false;
+
+        /* Check if the scope has any route-only domains except of "~.", i. e. whether it should only be used for particular
```

Comment:
except for ...

Those comments are also annoying wide.

Reply:
(this comment is actually removed later on again in the same PR...)